src/SourceBuild/content/Directory.Build.props

```diff
@@ -6,6 +6,18 @@
     <DotNetBuildOrchestrator>true</DotNetBuildOrchestrator>
   </PropertyGroup>
 
+  <PropertyGroup Label="ShortStacks">
```

Comment:
Please add a comment above the property group explaining what a short stack is. I imagine that this might change in the future when i.e. aspnetcore starts producing an android runtime pack.